data-sources/elixir/config.yml

```diff
@@ -3,6 +3,9 @@ displayName: Elixir agent installation
 description: Start monitoring the performance of an Elixir application or service by installing the New Relic Elixir agent.
 install:
   primary:
-    link:
-      url: https://docs.newrelic.com/docs/integrations/open-source-telemetry-integrations/elixir/elixir-open-source-agent
+    type: nerdlet
+    nerdlet_id: marketplace.install-data-source
+    nerdlet_state:
+      dataSourceId: elixir
+    requires_account: false
```

Comment:
```suggestion
    nerdlet: 
    nerdletId: marketplace.install-data-source
    nerdletState:
      dataSourceId: elixir
    requiresAccount: false
```

Reply:
Here is how you format data source nerdlet navigation: https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#data-source-install-fields

it is different than the way we format the internal data sources.